app/components/notification_message_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-notification-banner govuk-notification-banner--<%= notification_type %>" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      <%= title %>
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      <%= heading %>
+    </p>
+    <p class="govuk-body">
+      <% if message.present? %>
+        <%= message %>
```

Comment:
I was wondering how we would include a message which has a nested link embedded similar to https://design-system.service.gov.uk/components/notification-banner/success/index.html? (maybe we don't have that use case)

Reply:
We don't currently have that use case. An easy fix would potentially be to use `raw` or `html_safe` but I think we should not jump into this until required and then try and implement a more thought of solution.